### PR TITLE
Develop->master to deploy on next

### PR DESCRIPTION
**Pull request by kkellerlbl**
Also to be able to pull data_source_config.json to appdev and prod.

### CODE REVIEW

**Comment by kkellerlbl** [comment]
@briehl does this look reasonable?  If not I can always get data_source_config.json from the develop branch.

**Comment by kkellerlbl** [comment]
I'm planning on pushing up the existing narrative image on next before rebuilding, so this PR wouldn't get pushed up to production.

File: kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeManagePanel.js [hunk 1]
```diff
@@ -34,9 +34,9 @@ define([
 ) {
     'use strict';
     return new KBWidget({
-        name: "kbaseNarrativeManagePanel",
+        name: 'kbaseNarrativeManagePanel',
```
Comment (kkellerlbl) [inline kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeManagePanel.js]: This is the only diff that concerns me; there's a lot in here.  But if it seems okay let's just merge it.

Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeManagePanel.js]: That was where all the work was done to fix the bug where the wrong narrative was being copied. Also, some minor refactor and cleanup. It should be safe - there have been no complaints from CI or Next.

**Pull request merged**